### PR TITLE
Add command cooldowns function

### PR DESCRIPTION
**Pull request by ting-c**
Added command cooldowns function on root directory.
Implemented on covid-stats and mask commands currently.

### CODE REVIEW

File: commands/COVID/covid-stats.js [hunk 1]
```diff
@@ -49,31 +45,36 @@ module.exports = {
...
-						.setTitle(":octagonal_sign:  No Argument Given  :octagonal_sign:")
-						.setColor(colors.heptagram)
-						//const { prefix, token, roles, MongoDB, serverId, colors } = require('../../config.json');
-						.setDescription(`Oh no! You didn’t specify a country. Type \`${prefix}covid-stats list\` for the list of available countries`)
-					message.channel.send(noargembed);
+							.setTitle(
+								":octagonal_sign:  No Argument Given  :octagonal_sign:"
+							)
+							.setColor(colors.heptagram)
+							//const { prefix, token, roles, MongoDB, serverId, colors } = require('../../config.json');
```
Comment (j-dogcoder) [inline commands/COVID/covid-stats.js:57]:
```suggestion
							const { prefix, token, roles, MongoDB, serverId, colors } = require('../../config.json');
```

**Review by j-dogcoder — CHANGES_REQUESTED** [review]

**Comment by j-dogcoder** [comment]
Looks good overall @ting-c!

File: commands/COVID/covid-stats.js (hunk 1 above)
Comment (ting-c) [reply commands/COVID/covid-stats.js:57]: The commented code was there when pulled. It has to be removed or stay commented as it is located between chained method calls of a function. Would you like me to remove it?

Comment (j-dogcoder) [reply commands/COVID/covid-stats.js:57]: yes please!

**Review by j-dogcoder — APPROVED** [review]

**Pull request merged**